### PR TITLE
popup window

### PR DESCRIPTION
**Pull request by Lucas-Erkana**
✅ Added functionality that allows the user to click (or tap) the button to check project details, and the popup with details about the project appears.
✅ Added functionality that allows the user to click (or tap) the close (X) button, and the popup disappears.
✅ Added the following pieces of data in a JavaScript object: name, description, featured image, technologies, link to live version, link to source.
✅Add a javascript functionality that will update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. 
✅Add javascript that load cards section in HTML.
✅ Implement the popup window with javascript
✅ Implemented popups for both mobile and desktop screen sizes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,63 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Porfolio
```
Comment (HAMMAS-SALEEM) [inline README.md:3]: - [x] Your linters are not passing all the tests 😨 Kindly make your linters pass all the tests and submit your pull request again from your dashboard 👍

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Review Request 🔴

## Hey @Lucash2022, @brainskev :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github flow
✔️ Following some requirements

Good job on implementing some of the requirements, but i cannot proceed review on this pull request because there are some issues that you still need to work on :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Lucas-Erkana) [reply README.md:3]: @HAMMAS-SALEEM 
The problem above was [resolved](https://github.com/Lucash2022/LucasErkanaPortfolio_Website/commit/63033c8233bef57703eb51cf558144ddb61367d7).

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️

Hi @Lucash2022, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**